doc/material/v1.1.0.rst

```diff
@@ -0,0 +1,83 @@
+.. _whatsnew_0100:
+
+Version 1.1.0 (August 27, 2024)
+----------------------------------
```

Comment:
```suggestion
Version 1.1.0 (August 27, 2024)
-------------------------------
```

The lines underneath headings should be equally long as the headings. Please also check below.